packages/newspringchurchapp/src/tabs/connect/ActionBar.js

```diff
@@ -1,36 +1,41 @@
 import React from 'react';
-import Config from 'react-native-config';
 import { ActionBar, ActionBarItem } from '@apollosproject/ui-kit';
 import { withNavigation } from 'react-navigation';
 import PropTypes from 'prop-types';
+import { UserWebBrowserConsumer } from 'newspringchurchapp/src/user-web-browser';
 import { WebBrowserConsumer } from 'newspringchurchapp/src/ui/WebBrowser';
 
-const Toolbar = ({ navigation }) => (
-  <WebBrowserConsumer>
-    {(openUrl) =>
-      Config.EXPERIMENTAL === 'true' ? (
-        <ActionBar>
-          <ActionBarItem
-            onPress={() => navigation.navigate('Passes')}
-            icon="check"
-            label="Check-in"
-          />
-          <ActionBarItem
-            onPress={() =>
-              openUrl('https://my.newspring.cc', { externalBrowser: true })
-            }
-            icon="download"
-            label="Give"
-          />
-          <ActionBarItem
-            onPress={() => navigation.navigate('TestingControlPanel')}
-            icon="information"
-            label="Test"
-          />
-        </ActionBar>
-      ) : null
-    }
-  </WebBrowserConsumer>
+const Toolbar = () => (
+  <ActionBar>
+    <WebBrowserConsumer>
+      {(openUrl) => (
+        <ActionBarItem
+          onPress={() =>
+            openUrl(
+              'https://newspring.cc/give/now1',
+              { externalBrowser: true },
+              { useRockToken: true }
+            )
+          }
+          icon="give"
+          label="Give"
+        />
+      )}
+    </WebBrowserConsumer>
+    <UserWebBrowserConsumer>
+      {(openUserWebView) => (
+        <ActionBarItem
+          onPress={() =>
+            openUserWebView({
+              url: 'https://newspring.cc/serving',
```

Comment:
let's use the inappbrowser here with the rock token

Reply:
Figured out the in-app/external browser thing.